distributed/distributed.yaml

```diff
@@ -149,6 +149,12 @@ distributed:
       # Set to false for no maximum.
       max-spill: false
 
+      monitor-interval:
+        # Interval between checks for the spill, pause, and terminate thresholds.
+        # The target threshold is checked every time new data is inserted.
+        spill-pause: 200ms  # memory monitor on the Worker
+        terminate: 100ms  # memory monitor on the Nanny
```

Comment:
I've been considering merging these two for the sake of simplicity. Both at 200ms? Both at 100ms?  Please share your opinion.

Reply:
Merged at 100ms